operator/src/test/java/oracle/kubernetes/operator/helpers/KubernetesUtilsTest.java

```diff
@@ -16,34 +18,40 @@
 
   @Test
   public void whenCreationTimesDiffer_metadataWithLaterTimeIsNewer() {
-    V1ObjectMeta meta1 = new V1ObjectMeta().creationTimestamp(new DateTime(2)).resourceVersion("2");
-    V1ObjectMeta meta2 = new V1ObjectMeta().creationTimestamp(new DateTime(1)).resourceVersion("1");
+    V1ObjectMeta meta1 = new V1ObjectMeta().creationTimestamp(
```

Comment:
This is now more difficult to read than before. I'd recommend extracting  a method, such as:
```
private OffsetDateTime createTimeStamp(long time) {
    return OffsetDateTime.ofInstant(Instant.ofEpochSecond(time), ZoneId.of("UTC"));
}
``` 
and use that in these expressions.